flake.nix

```diff
@@ -13,7 +13,7 @@
         devShells.default = pkgs.mkShell
           {
             buildInputs = with pkgs;[
-              jdk8
+              jdk11
```

Comment:
@wing328 the `run-in-docker.sh` also uses jdk11, so this change looks good to me, any veto against this change in the common area?

Reply:
Exactly. Recent maven versions require JDK >= 11.